src/index.ts

```diff
@@ -195,11 +185,15 @@ function cosmiconfigSync(
   };
 }
 
+function normalizeOptions(
+  moduleName: string,
+  options: Options,
+): ExplorerOptions;
```

Comment:
This overload seems useless, since it has the same signature as the implementation.

Reply:
Yes, it's still a draft. Working on it